.circleci/ci-scripts/ci-mock-google-services-setup.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
```

Comment:
This file is a copy of [this one](https://github.com/squanchy-dev/squanchy-android/blob/develop/.circleci/ci-scripts/ci-mock-google-services-setup.sh) but adapted to also output a dummy plist for the iOS app